aws/test_ec2_ebs_volume_attached_to_instance.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from helpers import get_param_id
+
+from aws.ec2.resources import ec2_ebs_volumes
+from aws.ec2.helpers import ebs_volume_attached_to_instance
+
+
+@pytest.mark.ec2
+@pytest.mark.parametrize(
+    "ec2_ebs_volume", ec2_ebs_volumes(), ids=lambda ebs: get_param_id(ebs, "VolumeId")
+)
+def test_ec2_ebs_volume_attached_to_instance(ec2_ebs_volume):
```

Comment:
@bqbn Would you mind adding a short description in the docstring, as well including that there is a 90 day grace period for the EBS volumes?

Reply:
Done.